dist/companies.json

```diff
@@ -8317,8 +8317,8 @@
             "description": "We create touch points throughout the consumer buying cycle. Through our digital expertise, proprietary assets, and deeply integrated partnerships, we close the loop between consumers and the brands or services they need."
         },
         "reddit": {
-            "name": "reddit",
-            "websiteUrl": "http://reddit.com",
+            "name": "Reddit, Inc.",
+            "websiteUrl": "www.reddit.com",
```

Comment:
full website address: `https://www.reddit.com/`
file for changes: `adguard_companies.json`